docs/changelog.md

```diff
@@ -33,11 +33,16 @@
 # Changelog
 
 ## Changes staged on develop
-### ATT&CK Workbench version 1.1.0
-#### Improvements in 1.1.0
-- Added validation for missing ATT&CK IDs on objects that support them. Validation for missing ATT&CK IDs was also added when creating a collection. See [frontend#231](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/issues/231).
-
-#### Fixes in 1.1.0
+### ATT&CK Workbench version 1.0.3
+#### Improvements in 1.0.3
+- Added object type documentation on list pages. See [frontend#221](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/issues/221).
+- Improved the visibility of errors in collection imports:
+    - The user will now be provided with a downloadable list of objects that could not be saved (and the reason why) in the event of import errors.
+    - REST API will now log import errors to the console.
+    - Frontend will now log import errors to the console when the application environment is not set to production.
+- Added validation for missing ATT&CK IDs on objects that support them. `StixObject` class now includes boolean field (`supportsAttackID`) that specifies whether they support ATT&CK IDs or not. Inheriting classes must specify this field. Validation for missing ATT&CK IDs was also added when creating a collection. See [frontend#231](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/issues/231).
```

Comment:
Typically changelogs for this kind of application are for the most part targeted towards users, not developers. I think this particular entry is too technical for the audience of the changelog. Recommend changing to:

> The user will now be warned if they neglect to assign an ATT&CK ID to an object which supports it. When exporting a collection, the user will similarly be warned if any contained objects are missing ATT&CK IDs.

Reply:
Or just what you had before would also be fine:

> Added validation for missing ATT&CK IDs on objects that support them. Validation for missing ATT&CK IDs was also added when creating a collection. See [frontend#231](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/issues/231).